packages/components/inputs/checkbox-input/src/checkbox-input.styles.ts

```diff
@@ -1,59 +0,0 @@
-import { css } from '@emotion/react';
```

Comment:
I was wondering about this file removal.
It seems we now have several styling code in the component file itself (`getBorderColor`, `getBackgroundColor`, `getTextColor`, `CheckboxIcon` styles) and I thought they would be a good fit for a styles file.
Do we want to avoid this kind of styles files?